osimpipeline/task.py

```diff
@@ -96,29 +105,183 @@ def __init__(self, trial):
         super(TrialTask, self).__init__(trial.subject)
         self.trial = trial
 
+class SetupTask(TrialTask):
+    def __init__(self, tool, trial, 
+                 model_to_adjust=None, 
+                 adjusted_model=None,
+                 gait_cycles='concatenated'):
+        super(SetupTask, self).__init__(trial)
+        self.tool = tool
+        self.trial = trial
+        self.name = '%s_%s_setup' % (trial.id, self.tool)
+        self.tool_path = os.path.join(trial.results_exp_path, self.tool)
+        self.doit_path = self.study.config['doit_path']
+        self.gait_cycles=gait_cycles
+        self.source_extloads_fpath = os.path.join(self.trial.rel_path, 
+                self.tool, 'external_loads.xml')
```

Comment:
I think there might be a bug where the external loads file needs to be in the same directory as the setup file to avoid misinterpreting relative paths (relative paths in the external loads file are interpreted relative to the setup file rather than as relative to the external loads file). Let me know if you run into this.